### PR TITLE
RSS Feed for Blog

### PR DESCRIPTION
**Pull request by adriantirusli**
In this update I add `build-blog.js`. I think there will be a lot of rubbish in other files but please feel free to clean it lol. So sorry this took > a month 👉🏼👈🏼

### CODE REVIEW

**Comment by faultable** [comment]
Fimally! Will take a look soon

Anyway, good job!

File: build-blog.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                .split('.')
+                .slice(0, -1)
+                .join('.')
+            }
+
+            datas.push(data)
+
+            datas.forEach((data, index) => {
+              Object.assign(data.attributes, files[index])
+            })
```
Comment (faultable) [inline build-blog.js:48]: Di baris ini dia ngapain? Just curious

File: next.config.js [hunk 2]
```diff
@@ -6,5 +6,12 @@ const withMDX = require('./lib/mdx-loader')({
...
 
 module.exports = withMDX({
-  pageExtensions: ['md', 'mdx', 'jsx', 'js']
+  pageExtensions: ['md', 'mdx', 'jsx', 'js'],
+  webpack: config => {
+    return Object.assign({}, config, {
+      node: {
+        fs: 'empty'
+      }
+    })
```
Comment (faultable) [inline next.config.js:15]: Kenapa kita butuh ini?

File: package.json [hunk 3]
```diff
@@ -27,7 +29,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged"
+      "pre-commit": "pretty-quick --staged",
+      "pre-push": "yarn test"
```
Comment (faultable) [inline package.json:33]: Mungkin `pre-push` nya `yarn export` kali ya?

```suggestion
      "pre-push": "yarn export"
```

File: next.config.js (hunk 2 above)
Comment (adriantirusli) [reply next.config.js:15]: Kyknya ga butuh sih, told you banyak rubbish 🙈

File: build-blog.js (hunk 1 above)
Comment (adriantirusli) [reply build-blog.js:48]: Buat linknya

File: package.json (hunk 3 above)
Comment (adriantirusli) [reply package.json:33]: Hooh

File: next.config.js (hunk 2 above)
Comment (faultable) [reply next.config.js:15]: Never mind, take it easy

File: build-blog.js (hunk 1 above)
Comment (faultable) [reply build-blog.js:48]: I see, thank you!

File: next.config.js [hunk 4]
```diff
@@ -6,5 +6,12 @@ const withMDX = require('./lib/mdx-loader')({
...
+  }
```
Comment (faultable) [inline next.config.js:16]: Does this look good?

```suggestion
  pageExtensions: ['md', 'mdx', 'jsx', 'js']
```

File: package.json (hunk 3 above)
Comment (faultable) [reply package.json:33]: Nice, commit the suggestion then!

File: next.config.js (hunk 4 above)
Comment (adriantirusli) [reply next.config.js:16]: Yup, lgtm

**Pull request merged**